### ISSUE TITLE
parcels_get_examples on windows

### ISSUE THREAD

**Issue by delandmeterp** [OceanParcels/parcels#533]
Hi @willirath,

All the students installing Parcels on a Windows machine are coming to me with the same problem. They can't download the examples using the command:
`parcels_get_examples parcels_examples`

So far, I was bypassing the problems by running manually `python get_examples.py example_dir` within the env directory of conda. There should be a correct way to run this, no? Do you know what am I missing ?

Thanks!

**Comment by willirath** [comment]
I'm not sure. How is this done on appveyor?

**Comment by willirath** [comment]
I'd think that https://github.com/OceanParcels/parcels/blob/master/appveyor.yml has all necessary steps. But it builds and installs locally and not from the conda-forge package.

Maybe the `console_script` part in the setup.py is not properly handled in the conda build?

**Comment by delandmeterp** [comment]
Yes indeed. In the automatic tests, they install parcels dependencies, checkout last version, install the setup.py that write an exe file into a build directory.

So conda does not run any `setup.py install` ?

**Comment by willirath** [comment]
Steps I'd take to Isolate the problem:

- Print contents of the `parcels_get_examples` script on appveyor and check against the same on a local Windows install of Parcels.

- Add running `parcels_get_examples` as a test to the conda-forge recipe. (Then, we'd see if this functionality is broken already in the conda-forge package.)

**Comment by willirath** [comment]
> So conda does not run any setup.py install ?

 It does. Or it uses pip. I'm not sure.

**Comment by willirath** [comment]
See https://github.com/conda-forge/parcels-feedstock/blob/master/recipe/meta.yaml

On conda forge, a wheel is built with the setup.py and then this is installed with pip.

**Comment by willirath** [comment]
So let's find out how to add a test to the recipe that runs `parcels_get_examples` on the package and see if this breaks. (I can't be of much help / can't try anything myself before some time next week, though.)

**Comment by delandmeterp** [comment]
Thanks! I haven't fully understood all the processes within the meta.yaml, but now I know where to search! I'll have a look in the next few days and come back here.

**Comment by willirath** [comment]
See https://conda-forge.org/docs/testing.html#simple-existence-testing-example for how to add arbitrary commands to the test section of the meta.yaml

**Comment by erikvansebille** [comment]
I just checked on a windows machine, and the error is 
```
Fatal error in launcher: Unable to create process using
'"c:\bld\parcels_1545306932520\_h_env\python.exe"
"C:\Users\erik\Miniconda2\envs\parcelsv2\Scripts\parcels_get_examples.exe" '
```
Googling didn't really help, I fear. Much about spaces in directory names (e.g. https://github.com/pypa/pip/issues/2783), but that doesn't seem to be the issue here

The good news, though, is that I found a relatively easy workaround
1) Run `curl https://raw.githubusercontent.com/OceanParcels/parcels/master/parcels/scripts/get_examples.py > parcels_get_examples.py`
2) Run `python parcels_get_examples.py parcels_examples`

Shall we add a note to the installation instructions on oceanparcels.org to suggest this workaround on Windows?
